src/test/scala/com/intenthq/action_processor/integrations/Defaults.scala

```diff
@@ -0,0 +1,22 @@
+package com.intenthq.action_processor.integrations
+
+import java.nio.file.Paths
+
+import com.intenthq.action_processor.integrations.config.MapDbSettings
+import com.intenthq.action_processor.integrations.feeds.{FeedContext, FeedFilter}
+
+object Defaults {
+
+  val mapDbSettings: MapDbSettings = MapDbSettings(
+    dbPath = Paths.get("/tmp"),
+    startDbSize = 1L * 1024 * 1024,
```

Comment:
Are we using 1MB of initial MapDB allocated file for tests?
Don't you think for the type of tests we do maybe 4KB of initial size + increments of +4KB  or even less could we enough?

Reply:
I changed to 4KB init and increment size